src/test/regress/expected/sequencenes_owned_by.out

```diff
@@ -0,0 +1,314 @@
+                        result
+---------------------------------------------------------------------
+ ERROR:  relation "sequence tests.sc 1" does not exist
+(1 row)
+
+-- Citus local tables handles sequences slightly differently, so lets have one complex example
+-- which is combination of all the examples above
+CREATE SEQUENCE "sc 1";
+CREATE SEQUENCE "sc 2";
+CREATE SEQUENCE "sc 3";
+CREATE TABLE date_partitioned_citus_local_table_seq( measureid bigserial, col_a bigint, col_b bigserial, eventdate date, measure_data jsonb, PRIMARY KEY (measureid, eventdate)) PARTITION BY RANGE(eventdate);
+SELECT create_time_partitions('date_partitioned_citus_local_table_seq', INTERVAL '1 month', '2022-01-01', '2021-01-01');
+ create_time_partitions
+---------------------------------------------------------------------
+ t
+(1 row)
+
+ALTER SEQUENCE "sc 1" OWNED BY date_partitioned_citus_local_table_seq.col_a;
+ALTER SEQUENCE "sc 2" OWNED BY date_partitioned_citus_local_table_seq.col_a;
+ALTER SEQUENCE "sc 3" OWNED BY date_partitioned_citus_local_table_seq.col_b;
+ALTER SEQUENCE "sc 2" OWNED BY date_partitioned_citus_local_table_seq.col_b;
+SELECT citus_add_local_table_to_metadata('date_partitioned_citus_local_table_seq');
+ citus_add_local_table_to_metadata
+---------------------------------------------------------------------
+
+(1 row)
+
+-- show that "sc 1", "sc 2" and test_d_seq are distributed
+SELECT DISTINCT result FROM run_command_on_workers($$SELECT '"sequence tests"."sc 1"'::regclass::text || ',' || '"sequence tests"."sc 2"'::regclass::text ||  ',' || '"sequence tests"."sc 3"'::regclass::text || ',' || '"sequence tests"."date_partitioned_citus_local_table_seq_col_b_seq"'::regclass::text; $$);
+                                                                  result
+---------------------------------------------------------------------
+ "sequence tests"."sc 1","sequence tests"."sc 2","sequence tests"."sc 3","sequence tests".date_partitioned_citus_local_table_seq_col_b_seq
+(1 row)
+
+SELECT
+	pg_identify_object_as_address(classid, objid, objsubid)
+FROM pg_dist_object WHERE classid = 1259 AND (objid IN ('"sequence tests"."sc 1"'::regclass, '"sequence tests"."sc 2"'::regclass, '"sequence tests"."sc 3"'::regclass, '"sequence tests"."date_partitioned_citus_local_table_seq_col_b_seq"'::regclass)) ORDER BY 1;
+                             pg_identify_object_as_address
+---------------------------------------------------------------------
+ (sequence,"{""sequence tests"",date_partitioned_citus_local_table_seq_col_b_seq}",{})
+ (sequence,"{""sequence tests"",""sc 1""}",{})
+ (sequence,"{""sequence tests"",""sc 2""}",{})
+ (sequence,"{""sequence tests"",""sc 3""}",{})
+(4 rows)
+
+-- this is not supported for Citus local tables as well, one day we might relax
+ALTER SEQUENCE "sc 2" OWNED BY date_partitioned_citus_local_table_seq.col_a;
+ERROR:  Altering a distributed sequence is currently not supported.
```

Comment:
We could in the future consider supporting `ALTER SEQUENCE .. OWNED BY table.col` on distributed in the future, and this PR seems like a good step for that.